lib/src/components/NxFormSelect/NxFormSelect.scss

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
Note: this file is mostly copied from `_nx-form-select.scss`, but also rearranged as needed for the new more complex structure